### PR TITLE
I2p - updated version -> 0.9.23

### PR DESCRIPTION
**Pull request by ephexeve**

### CODE REVIEW

File: Library/Formula/i2p.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class I2p < Formula
-  desc "I2P is an anonymous overlay network - a network within a network"
+  desc "An anonymous overlay network - a network within a network"
   homepage "https://geti2p.net"
-  url "https://download.i2p2.de/releases/0.9.21/i2pinstall_0.9.21.jar"
-  sha256 "0238ffc6ea44099ef4fe6c20913cb4eec675c2760aea07dfe7d499addcc89cf2"
+  url "https://dl.dropboxusercontent.com/u/18621288/I2P/0.9.23/i2pinstall_0.9.23.jar"
```
Comment (DomT4) [inline Library/Formula/i2p.rb]: Why aren't we using the `https://download.i2p2.de/releases/0.9.23/i2pinstall_0.9.23.jar` URL here? Seems to work okay.

Comment (ephexeve) [inline Library/Formula/i2p.rb]: @DomT4, weird, it wasn't working for me yesterday, and they have that URL as a mirror. I created a new commit with the new URL.

**Comment by mikemcquaid** [comment]
Test is hanging and failing: https://travis-ci.org/Homebrew/homebrew/jobs/94874348#L315

**Comment by ephexeve** [comment]
@mikemcquaid Hm, weird, testing the same commit locally seems to work fine. if I run a i2prouter start, it works fine too, status gives me "running".
If I run brew test i2p, it fails due permission problems. 

I don't have time to look at this now, will do it by the end of the week.

File: Library/Formula/i2p.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class I2p < Formula
-  desc "I2P is an anonymous overlay network - a network within a network"
+  desc "An anonymous overlay network - a network within a network"
```
Comment (bfontaine) [inline Library/Formula/i2p.rb]: Please remove the indefinite article: `desc "Anonymous ...`

**Comment by magicgoose** [comment]
FYI, I have the same test error with current i2p version from brew (0.9.21).

**Comment by bfontaine** [comment]
Anything new here? The latest release is now 0.9.25.

**Comment by ephexeve** [comment]
Moving this issue to `core` and opening a new one for version 0.9.25.

**Pull request closed without merging**